### ISSUE TITLE
support for https sso server urls

### ISSUE THREAD

**Issue by jagadishrajr** [divio/django-simple-sso#44]
My SSO server is behind an nginx reverse proxy with SSL enabled(self signed). My SSO clients are failing with this error.  

`Max retries exceeded with url: /server/request-token/ (Caused by SSLErILED] certificate verify failed: self signed certificate (_ssl.c:1076)')))`

What can I do to make this connection possible? How can I pass certificate when I call SSO server?

**Comment by FinalAngel** [comment]
@victor-yunenko any insights from your end?

**Comment by RafaelFigueiredo** [comment]
@jagadishrajr are you using a self signed certificate on your nginx?

**Comment by jagadishrajr** [comment]
@RafaelFigueiredo Yes, I am using self signed certificate. This worked when I created a rootCA for myself and used it to generate a signed certificate.
